packages/web/src/components/result/addons/Pagination.js

```diff
@@ -65,104 +65,108 @@ const buildPaginationDOM = (props, position) => {
-	}
-	return (
-		<div className={`${pagination} ${getClassName(innerClass, 'pagination')}`}>
-			<Button
-				className={getClassName(innerClass, 'button') || null}
-				disabled={currentPage === 0}
-				onKeyPress={event => handleA11yAction(event, onPrevPage)}
-				onClick={onPrevPage}
-				tabIndex="0"
-				{...prevHrefProp}
-			>
-				Prev
-			</Button>
-			{
+		if (currentPage >= 1) {
+			prevHrefProp = {
+				href: `?${fragmentName}=${currentPage}`,
+				rel: 'prev',
+			};
+		}
+
+		if (currentPage < totalPages - 1) {
+			nextHrefProp = {
+				href: `?${fragmentName}=${currentPage + 2}`,
+				rel: 'next',
+			};
+		}
+		return (
+			<div className={`${pagination} ${getClassName(innerClass, 'pagination')}`}>
 				<Button
-					className={className}
-					primary={primary}
-					onKeyPress={event => handleA11yAction(event, () => setPage(0))}
-					onClick={(e) => {
-						e.preventDefault();
-						setPage(0);
-					}}
+					className={getClassName(innerClass, 'button') || null}
+					disabled={currentPage === 0}
+					onKeyPress={event => handleA11yAction(event, onPrevPage)}
+					onClick={onPrevPage}
 					tabIndex="0"
-					href={`?${fragmentName}=1`}
+					{...prevHrefProp}
 				>
-					1
+					Prev
 				</Button>
-			}
-			{showEndPage && currentPage >= Math.floor(pages / 2) + !!(pages % 2)
-				? <span>...</span> : null}
-			{currentPage <= (totalPages - pages) + 2 && buildPaginationDOM(props, 'start')}
-			{showEndPage && pages > 2 && currentPage <= totalPages - Math.ceil(pages * 0.75) ? (
-				<span>...</span>
-			) : null}
-			{showEndPage && buildPaginationDOM(props, 'end')}
-			<Button
-				className={getClassName(innerClass, 'button') || null}
-				disabled={currentPage >= totalPages - 1}
-				onKeyPress={event => handleA11yAction(event, onNextPage)}
-				onClick={onNextPage}
-				tabIndex="0"
-				{...nextHrefProp}
-			>
-				Next
-			</Button>
-		</div>
-	);
+				{
+					<Button
+						className={className}
+						primary={primary}
+						onKeyPress={event => handleA11yAction(event, () => setPage(0))}
+						onClick={(e) => {
+							e.preventDefault();
+							setPage(0);
+						}}
+						tabIndex="0"
+						href={`?${fragmentName}=1`}
+					>
+						1
+					</Button>
+				}
+				{showEndPage && currentPage >= Math.floor(pages / 2) + !!(pages % 2) ? (
+					<span>...</span>
+				) : null}
+				{currentPage <= totalPages - pages + 2 && buildPaginationDOM(this.props, 'start')}
+				{showEndPage && pages > 2 && currentPage <= totalPages - Math.ceil(pages * 0.75) ? (
+					<span>...</span>
+				) : null}
+				{showEndPage && buildPaginationDOM(this.props, 'end')}
+				<Button
+					className={getClassName(innerClass, 'button') || null}
+					disabled={currentPage >= totalPages - 1}
+					onKeyPress={event => handleA11yAction(event, onNextPage)}
+					onClick={onNextPage}
+					tabIndex="0"
+					{...nextHrefProp}
+				>
+					Next
+				</Button>
+			</div>
+		);
+	}
 }
 
+export default Pagination;
+
 Pagination.propTypes = {
```

Comment:
@jyash97 Can you put that block before export statement?

Reply:
Okay